src/internal.rs

```diff
@@ -369,7 +369,7 @@ pub struct Rav1dFrameContext_frame_thread {
     pub b: Vec<Av1Block>,
     pub cbi: Vec<CodedBlockInfo>,
     // indexed using (t->by >> 1) * (f->b4_stride >> 1) + (t->bx >> 1)
-    pub pal: *mut [[u16; 8]; 3], /* [3 plane][8 idx] */
+    pub pal: Vec<[[u16; 8]; 3]>, /* [3 plane][8 idx] */
```

Comment:
Do we need an explicit alignment on this? We're replacing `alloc_aligned` with the vector allocation.

Reply:
```suggestion
    pub pal: Vec<Align64<[[u16; 8]; 3]>>, /* [3 plane][8 idx] */
```

Reply:
Ah good catch! Fixed.